src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -0,0 +1,51 @@
+package com.bardsoftware.backend.fleet.rmarkdown
+
+import com.google.cloud.ServiceOptions
+import com.google.cloud.pubsub.v1.AckReplyConsumer
+import com.google.cloud.pubsub.v1.MessageReceiver
+import com.google.cloud.pubsub.v1.Subscriber
+import com.google.pubsub.v1.PubsubMessage
+import com.google.pubsub.v1.SubscriptionName
+import java.util.concurrent.LinkedBlockingDeque
+
+object SubscriberExample {
+    private const val subscriberName = "rmarkdown-compiler"
+
+    // use the default project id
+    private val PROJECT_ID = ServiceOptions.getDefaultProjectId()
+
+    private val messages = LinkedBlockingDeque<PubsubMessage>()
+
+    internal class MessageReceiverExample : MessageReceiver {
+
+        override fun receiveMessage(message: PubsubMessage, consumer: AckReplyConsumer) {
+            messages.offer(message)
+            consumer.ack()
+        }
+    }
+
+    /** Receive messages over a subscription.  */
+    @Throws(Exception::class)
```

Comment:
It is okay to remove this annotation. We do not really care what main method throws.